alonzo/impl/src/Cardano/Ledger/Alonzo/TxWitness.hs

```diff
@@ -343,17 +347,10 @@ instance
   where
   mempty = ScriptData mempty mempty mempty
 
-data EraIndependentScriptData
+type ScriptDataHash crypto = SafeHash crypto EraIndependentScriptData
 
-newtype ScriptDataHash crypto
-  = ScriptDataHash
-      (Hash.Hash (HASH crypto) EraIndependentScriptData)
-  deriving (Show, Eq, Ord, Generic)
-  deriving newtype (NFData, NoThunks)
-
-deriving newtype instance CC.Crypto crypto => ToCBOR (ScriptDataHash crypto)
-
-deriving newtype instance CC.Crypto crypto => FromCBOR (ScriptDataHash crypto)
+hashScriptData :: forall era. Era era => ScriptData era -> ScriptDataHash (Crypto era)
+hashScriptData x = hashAnnotated x
```

Comment:
At this point I feel we might as well drop this function and just use `hashAnnotated`